cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -1589,6 +1589,25 @@ pPoolIdOutputFormat =
     , Opt.value IdOutputFormatBech32
     ]
 
+-- | @pQueryOutputFormat kind@ is a parser to specify in which format
+-- to view some data (json or text). @kind@ is the kind of data considered.
+pQueryOutputFormat :: String -> Parser QueryOutputFormat
```

Comment:
```suggestion
pOutputFormat :: String -> Parser OutputFormat
```
How about generalizing it a bit? That would move us closer towards https://github.com/IntersectMBO/cardano-cli/issues/566

Reply:
The thing is, in this specific PR, the output can either be JSON or text. In other cases we have JSON, Yaml, or text. So this PR is not super general.

I propose to look how to generalize in a follow-up PR :slightly_smiling_face: